bundle/manifests/external-dns-operator.clusterserviceversion.yaml

```diff
@@ -419,8 +419,7 @@ spec:
           selector:
             matchLabels:
               name: external-dns-operator
-          strategy:
-            type: Recreate
+          strategy: {}
```

Comment:
Since it is going to default to rolling update, I think it is best to provide some explicit values that we know for sure won't cause issues.